src/main/java/com/statful/client/framework/springboot/processor/ProcessorMap.java

```diff
@@ -0,0 +1,49 @@
+package com.statful.client.framework.springboot.processor;
+
+import com.statful.client.framework.springboot.processor.processors.http.HttpRequestsProcessor;
+import com.statful.client.framework.springboot.processor.processors.system.*;
+import com.statful.client.framework.springboot.processor.processors.tomcat.HttpSessionsProcessor;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public abstract class ProcessorMap {
+
+    private static Map<String, MetricProcessor> processors;
```

Comment:
If each processor was a bean you could perhaps use injection for this, to make easier to test. It would also allow an easy way to turn on and off each processor